src/recipe.js

```diff
@@ -1,11 +1,12 @@
 class Recipe {
-  constructor(recipe, ingredientsData) {
+  constructor(recipe, ingredientsData, recipeData) {
     this.name = recipe.name;
     this.id = recipe.id;
     this.ingredients = recipe.ingredients;
     this.instructions = recipe.instructions;
     this.tags = recipe.tags;
     this.ingredientsData = ingredientsData;
+    this.recipeData = recipeData;
```

Comment:
Adding the recipeData as a property in the Recipe class is a good addition where the recipeData is more accessible when we need to look for a specific recipe.